theme/webpack.config.js

```diff
@@ -1,76 +1,62 @@
 const dotenv = require('dotenv');
-const proxy = require('http-proxy-middleware');
 const webpack = require('webpack');
+const { URL } = require('url');
 const { resolve } = require('path');
 const UglifyPlugin = require('uglifyjs-webpack-plugin');
-const WorkboxPlugin = require('@magento/workbox-webpack-plugin');
-const WriteFileWebpackPlugin = require('write-file-webpack-plugin');
 const configureBabel = require('./babel.config.js');
-const getMagentoEnv = require('./devtools/get-magento-env');
-const express = require('express');
-const {
-    WebpackMagentoRootComponentsChunksPlugin
-} = require('@magento/anhinga');
+const { Webpack: WebpackTools } = require('@jzetlen/pwa-buildpack');
```

Comment:
Believe Zetlen merged this

Reply:
Yeah, that was my bad. I pushed an update directly to master to fix it during Imagine.